connect_ext_ppr/models/deployment.py

```diff
@@ -70,6 +71,21 @@ class DeploymentRequest(Model):
         innerjoin=True,
     )
 
+    @property
+    def track_field(self):
+        return 'status'
+
+    @transition(target=STATUSES.aborting, sources=[STATUSES.pending, STATUSES.processing])
+    def aborting(self, tasks, by):
+        self.aborted_at = datetime.utcnow()
```

Comment:
This fields should be setup after the DR is moved to the aborted status

Reply:
Fixed!